### PR TITLE
Add data income routine

### PR DESCRIPTION
**Pull request by rafaellehmkuhl**

### CODE REVIEW

File: services/sats_comm/main.py [hunk 1]
```diff
@@ -45,6 +38,107 @@ def send_data_through_rockblock():
...
+def send_mavlink_message(message: Dict[str, Any]) -> None:
+    mavlink2rest_package = {
+        "header": {"system_id": 1, "component_id": 194, "sequence": 0},
+        "message": message,
+    }
+    requests.post("http://127.0.0.1:6040/mavlink", data=json.dumps(mavlink2rest_package), timeout=10.0)
+
+def deal_with_income_data(income_data: bytes) -> None:
+    if income_data.decode().startswith("waypoint"):
+        _, lat, lon, wait_time, next_id = income_data.decode().split(":")
```
Comment (joaoantoniocardoso) [inline services/sats_comm/main.py:113]: Note that this unpacking to 5 variables can fail if we have less or more than 4 ":", raising ValueError.

nevermind, I just saw that this function is being called from a try.

File: services/sats_comm/main.py [hunk 2]
```diff
@@ -45,6 +38,107 @@ def send_data_through_rockblock():
...
+    logger.debug("Modem information:")
+    logger.debug(f"Model: {rb.model}")
+    logger.debug(f"Revision: {rb.revision}")
+    logger.debug(f"Serial number: {rb.serial_number}")
+    logger.debug(f"Status: {rb.status}")
+    ser.close()
+
+def init_rockblock() -> Tuple[RockBlock, serial.Serial]:
+    ser = serial.Serial(
+        "/dev/serial/by-path/platform-fd500000.pcie-pci-0000:01:00.0-usb-0:1.1:1.0-port0",
```
Comment (joaoantoniocardoso) [inline services/sats_comm/main.py]: Can't we pass it as a CLI argument "--serial", like as in the other services?

File: services/sats_comm/main.py (hunk 1 above)
Comment (rafaellehmkuhl) [reply services/sats_comm/main.py:113]: You mean if we send the wrong message?
Should we put a logger for the exception?

File: services/sats_comm/main.py (hunk 2 above)
Comment (rafaellehmkuhl) [reply services/sats_comm/main.py]: Yup. Forgot to do it.
Fixed.

File: services/sats_comm/main.py (hunk 1 above)
Comment (joaoantoniocardoso) [reply services/sats_comm/main.py:113]: I think it is fine because the caller is using a try and is already debugging the exception =)

**Review by joaoantoniocardoso — APPROVED** [review]

**Review by patrickelectric — APPROVED** [review]

**Pull request merged**